### ISSUE TITLE
Can

### ISSUE THREAD

**Issue by LiothGold** [goatfungus/NMSSaveEditor#816]

**Comment by goatfungus** [comment]
Play the game in offline mode when you want to use any account unlocks, otherwise the game will reload your real settings from the cloud.

**Comment by LiothGold** [comment]
Not quite what I meant. Though that is part of it, as I can't actually 'claim' any of the account unlocks as it lists them all as 'claimed' already. But my issue is I can't actually 'use' the Quicksilver shop, I can't buy anything.

**Comment by LiothGold** [comment]
[Save.zip](https://github.com/goatfungus/NMSSaveEditor/files/11396876/Save.zip)
Here's the save file incase this helps!

**Comment by woodstockfurygit** [comment]
I wish somebody would explain what "play the game in offline mode" means... If it means that you choose "Play Game" instead of "Multiplayer" at the opening screen, this DOES NOT WORK. If it means something else, I wish whoever posts this language would explain how to "play in offline mode" because I can't see an option for that anywhere.

**Comment by LiothGold** [comment]
> I wish somebody would explain what "play the game in offline mode" means... If it means that you choose "Play Game" instead of "Multiplayer" at the opening screen, this DOES NOT WORK. If it means something else, I wish whoever posts this language would explain how to "play in offline mode" because I can't see an option for that anywhere.

If you're playing on Steam, play in 'offline' mode. Sign out of 'friends and chat'.

**Comment by Babscoole** [comment]
> I wish somebody would explain what "play the game in offline mode" means... If it means that you choose "Play Game" instead of "Multiplayer" at the opening screen, this DOES NOT WORK. If it means something else, I wish whoever posts this language would explain how to "play in offline mode" because I can't see an option for that anywhere.

It means while completely disconnected from the internet.  Turn off your wi-fi of unplug the ethernet cable from your computer.  There is also an option to play in

**Comment by SplinterGU** [comment]
> > I wish somebody would explain what "play the game in offline mode" means... If it means that you choose "Play Game" instead of "Multiplayer" at the opening screen, this DOES NOT WORK. If it means something else, I wish whoever posts this language would explain how to "play in offline mode" because I can't see an option for that anywhere.
> 
> It means while completely disconnected from the internet. Turn off your wi-fi or unplug the ethernet cable from your computer.

yes, I confirm... complete offline (internet disconnected) is required, and this must be disconnected before start the game... (and maybe steam, too)

**Comment by AlucardTheRed9** [comment]
> I Can't collect anything from the Quicksilver Synthesis shop even being off line. Also the game crashes. What can I do?

Idk, you have any mods? It works for me fine. I think you only need offline mode for twitch rewards. its either that, or maybe the save itself got messed up somehow.

**Comment by LiothGold** [comment]
I wasn't talking about the Expedition/Twitch rewards, I was talking about the fucking SHOP.

**Comment by GrumpyKoala1966** [comment]
Can we all be civil in chat, there is no need to use the "F" word.
